docs/src/blockchain-development/storage.md

```diff
@@ -6,6 +6,60 @@ Put in conventional programming terms, contract storage is like saving data to a
 
 Some basic use cases of storage include declaring an owner address for a contract and saving balances in a wallet.
 
+## Storage By-Hand
+
+Outside of the newer experimental `storage` syntax which is being stabalized, you can leverage FuelVM storage operations using the `store` and `get` methods provided in the standard (`std`) library.
+
+With this approach you will have to design the internal key and value design manually.
+
+An example is as follows:
+
+```sway
+contract;
+
+use std::{
+    storage::{get, store}
+};
+
+abi StorageExample {
+    fn store_something(amount: u64);
+    fn get_something() -> u64;
+}
+
+const STORAGE_KEY: b256 = 0x0000000000000000000000000000000000000000000000000000000000000000;
+
+impl StorageExample for Contract {
+    fn store_something(amount: u64) {
+        store(STORAGE_KEY, amount);
+    }
+
+    fn get_something() -> u64 {
+        let value = get::<u64>(STORAGE_KEY);
+        value
+    }
+}
+```
+
```

Comment:
Add disclaimer that this only works for primitive types up to a word, and add a link to #1229 which will be able to handle storage of any arbitrary stack type.

Reply:
@adlerjohn right, but currently that is not the case? I'm just trying to offer a developer something for now until we truly resolve this with whatever is the appropriate way to do it all in `store` and `get`.

Happy to link to an issue though, as that is more maintainable.

Reply:
Honestly I would simply finish #1229 first, then finish this PR without needing a disclaimer.

Reply:
@adlerjohn yeah, I mean I literally wrote that code in 3 minutes. Just add some tests and have it reviewed lol.

Reply:
Looks like #1229 will take some more time to be finished due to some compiler issues, so just add the disclaimer.

Reply:
Okay, I'll reference the error so people can find the code they need to unblock for now.

I agree, no STD for that stuff (it will be removed eventually), but I just want a quick thing I can point to for a work around.